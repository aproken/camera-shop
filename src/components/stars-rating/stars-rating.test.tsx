import { render, RenderResult } from '@testing-library/react';
import StarsRating from './stars-rating';

test('Тестирование компонента StarsRating', () => {
  const rating = 3.5;

  const { container }: RenderResult = render(
    <StarsRating rating={rating} />
  );

  expect(container).toMatchSnapshot();
});
