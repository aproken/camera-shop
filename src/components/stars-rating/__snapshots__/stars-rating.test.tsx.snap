// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Тестирование компонента StarsRating 1`] = `
<div>
  <div
    class="rate review-card__rate"
  >
    <svg
      aria-hidden="true"
      height="16"
      width="17"
    >
      <use
        xlink:href="#icon-full-star"
      />
    </svg>
    <svg
      aria-hidden="true"
      height="16"
      width="17"
    >
      <use
        xlink:href="#icon-full-star"
      />
    </svg>
    <svg
      aria-hidden="true"
      height="16"
      width="17"
    >
      <use
        xlink:href="#icon-full-star"
      />
    </svg>
    <svg
      aria-hidden="true"
      height="16"
      width="17"
    >
      <use
        xlink:href="#icon-star"
      />
    </svg>
    <svg
      aria-hidden="true"
      height="16"
      width="17"
    >
      <use
        xlink:href="#icon-star"
      />
    </svg>
    <p
      class="visually-hidden"
    >
      Оценка: 
      3.5
    </p>
  </div>
</div>
`;
